pkg/ebpf/tracee.go

```diff
@@ -125,6 +125,8 @@ type Tracee struct {
 	// This does not mean they are required for tracee to function.
 	// TODO: remove this in favor of dependency manager nodes
 	requiredKsyms []string
+	// Probes created for suspicious_syscall_source event
+	suspiciousSyscallSourceProbes *probes.ProbeGroup
```

Comment:
I don't think it is a good idea to add probe groups into the tracee struct.
It makes the struct coupled to specific events, and we want to decouple things, not the other way around.

Reply:
Where should it be added? The main probe group is also part of the tracee struct

Reply:
So where should I put it? I also plan to add additional "syscall checker" events that will all utilize the same probe group which will make it less not specific to a single event but for a "class" of events

Reply:
@yanivagman I added a map to the tracee struct that holds extra probe groups (indexed by a given name) that can be added dynamically.